codecov.yml

```diff
@@ -7,6 +7,8 @@ coverage:
       default: false
     patch:
       default: false
+  ignore:
+    - "*.Designer.cs"
```

Comment:
❓ Where are you seeing this? I just checked the latest coverage report and couldn't find any files with this pattern.

Reply:
When I wrote that I didn't see the exclusion performed in `appveyor.yml`.

This might be a more appropriate place to put it, so runs outside AppVeyor get identical results.

Reply:
This file is only used by codecov.yml. If we ever update the code to allow local coverage runs (e.g. using ReportBuilder), we'll want to use the approach in appveyor.yml so it applies to both local and only experiences.

Reply:
❕ It looks like these are already getting ignored, so this section can be removed.